components/user-mgt/org.wso2.carbon.identity.user.pre.update.profile.action/pom.xml

```diff
@@ -91,13 +121,20 @@
 
                             org.wso2.carbon.context; version="${carbon.kernel.package.import.version.range}",
                             org.wso2.carbon.utils; version="${carbon.kernel.package.import.version.range}",
+                            org.wso2.carbon.user.core.*;version="${carbon.kernel.package.import.version.range}",
+                            org.wso2.carbon.user.api; version="${carbon.user.api.imp.pkg.version.range}",
 
                             org.wso2.carbon.identity.core.context; version="${carbon.identity.package.import.version.range}",
                             org.wso2.carbon.identity.core.context.model; version="${carbon.identity.package.import.version.range}",
                             org.wso2.carbon.identity.core.util; version="${carbon.identity.package.import.version.range}",
                             org.wso2.carbon.identity.action.management.api.exception; version="${carbon.identity.package.import.version.range}",
                             org.wso2.carbon.identity.action.management.api.model; version="${carbon.identity.package.import.version.range}",
-                            org.wso2.carbon.identity.action.management.api.service; version="${carbon.identity.package.import.version.range}"
+                            org.wso2.carbon.identity.action.management.api.service; version="${carbon.identity.package.import.version.range}",
+                            org.wso2.carbon.identity.action.execution.api.*; version="${carbon.identity.package.import.version.range}",
+                            org.wso2.carbon.identity.user.action.api.service; version="${carbon.identity.package.import.version.range}",
```

Comment:
Are we using the `UserActionExecutor` in this component ?

Reply:
we are using the UserActionContext here

Reply:
It is in the model package, which we already added right ?

Reply:
ok noted

Reply:
Fixed in https://github.com/wso2/carbon-identity-framework/pull/6620/commits/d53630b2d21ccbd71a565fce33e9f06bdd68f46c